isb_web/sqlmodel_database.py

```diff
@@ -0,0 +1,141 @@
+import datetime
+import sqlalchemy
+from typing import Optional, List
+from sqlmodel import SQLModel, create_engine, Session, select
+from isb_lib.models.thing import Thing
+from isb_web.schemas import ThingPage
+
+
+class SQLModelDAO:
```

Comment:
New object to encapsulate the db connection, engine, and session creation.  It's the responsibility of callers to manage their own sessions.